src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import './HomePage.css'
+
+function HomePage() {
+  const [drinkType, setDrinkType] = useState('')
+  const navigate = useNavigate()
+
+  const handleChoice = (choice) => {
+    if (choice === 'alcoholic') {
+      navigate('/all')
+    } else if (choice === 'na') {
+      navigate('/na')
+    }
+  }
+
+  return (
+    <div>
+      <section className="home-title">
+        <h1>Mix Master</h1>
+        <p>your drink recipe companion</p>
+      </section>
+      <section className="quiz-section">
+        <h1 className="quiz-title">Do you prefer...</h1>
+        <div className="radio-container">
+          <label className="radio">
+              Alcoholic beverages
+            <input
+              type="radio"
+              name="drinkType"
+              value="alcoholic"
```

Comment:
I think I'll change the values to a boolean "true" or "false" so the state is reusable for the filter function

Reply:
that sounds like a good plan. you might have to update the handleClick function logic? 
excited to see what you come up with!